app/screens/AppNavigator/screens/Settings/SettingsScreen.tsx

```diff
@@ -34,12 +34,13 @@ export function SettingsScreen ({ navigation }: Props): JSX.Element {
     }
 
     const auth: Authentication<string[]> = {
-      message: translate('screens/Settings', 'To continue downloading your recovery words, we need you to enter your passcode.'),
       consume: async passphrase => await MnemonicStorage.get(passphrase),
       onAuthenticated: async (words) => {
         navigation.navigate({ name: 'RecoveryWordsScreen', params: { words }, merge: true })
       },
-      onError: e => Logging.error(e)
+      onError: e => Logging.error(e),
+      message: translate('screens/Settings', 'To continue downloading your recovery words, we need you to enter your passcode.'),
```

Comment:
Maybe it would be better if we don't use downloading as users are not downloading a copy. To continue viewing your recovery words...